### PR TITLE
Introduce address-with-name and address to replace full-address

### PR DESCRIPTION
**Pull request by battre**
This enables us to have a clearer representation of an address without a name and an address with a name.

### CODE REVIEW

File: model/countries/global/global-descriptions.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 short-descriptions:
-  full-address: Artificial concept, not to be used in HTML
+  address: Address of a physical location but without a name - Artificial concept, not to be used in HTML
```
Comment (norgevz) [inline model/countries/global/global-descriptions.yaml:2]: Can probably drop the "but without a name" part.

**Review by norgevz — APPROVED** [review]
lgtm

File: model/countries/global/global-descriptions.yaml (hunk 1 above)
Comment (norgevz) [reply model/countries/global/global-descriptions.yaml:2]: Once more change I remembered. The model for IN contains locality2 twice. It exist under `street-address-alternative-1` and `address`.

Comment (battre) [reply model/countries/global/global-descriptions.yaml:2]: Yes... the modeling for India will require more thoughts.

**Pull request merged**